### ISSUE TITLE
Old SRDB data

### ISSUE THREAD

**Issue by jinshijian** [jinshijian/ESSD#4]
@bpbond 
Can you please add srdb-v1 to v3 file to the data folder? Thanks

**Comment by bpbond** [comment]
Hi @jinshijian - the older versions of SRDB are I think all available at https://code.google.com/archive/p/srdb/downloads

**Comment by jinshijian** [comment]
Great, thanks! @bpbond 
Since there are more than 4 old versions, I chose:
srdb_20100209a.zip as srdb-v1 as it is the "First availability of full data set"
srdb_20120510a.zip as srdb-v2 (not sure, can change to another one)
srdb_20131218a.zip as srdb-v3 as it is the first release in GIthub repo
ORNL DAAC version 4 as srdb-v4 (obviously)
Please let me know if some of them are not a good pick, thanks

**Comment by bpbond** [comment]
Nope, those looks like good picks. Just make sure the methods section of the manuscript clearly specifies the above information.

**Comment by jinshijian** [comment]
Sure, it looks like there is not too much difference from v1 to v4. Any comments?

![image](https://user-images.githubusercontent.com/13302161/77366580-a9a04e80-6d2e-11ea-9f43-743c1d05f390.png)

**Comment by bpbond** [comment]
Another options would be use density plots - have the global data in a heavy line, and then each SRDB version is a separate line. With different ecosystem types in different facets?

**Comment by jinshijian** [comment]
Here is the density comparison, much better than box plot!
![image](https://user-images.githubusercontent.com/13302161/77871787-2e013e80-7213-11ea-97f5-2037b6c0cdb8.png)

**Comment by bpbond** [comment]
Oh that's cool! It looks like red is global, i.e. reality, and teal is SRDB? Really neat figure.

**Comment by jinshijian** [comment]
Study with Rh measurement
![image](https://user-images.githubusercontent.com/13302161/77917361-7a2a9e00-7268-11ea-8458-fd301555b712.png)

**Comment by jinshijian** [comment]
> Oh that's cool! It looks like red is global, i.e. reality, and teal is SRDB? Really neat figure.

Correct, red is global, and other one is SRDB
